data/presets/presets/place/welcome_sign.json

```diff
@@ -0,0 +1,15 @@
+{
+  "icon": "maki-lighthouse",
+  "geometry": [
+    "point",
+    "area"
```

Comment:
I think these tags are only used for points. Can you remove `area`?

Reply:
Corrected